src/main/java/nextstep/lms/domain/PricingPolicy.java

```diff
@@ -0,0 +1,23 @@
+package nextstep.lms.domain;
+
+import nextstep.lms.enums.PricingTypeEnum;
+
+public class PricingPolicy {
+    private final PricingTypeEnum pricingTypeEnum;
+    private final int tuitionFee;
+
+    public PricingPolicy(PricingTypeEnum pricingTypeEnum, int tuitionFee) {
+        this.pricingTypeEnum = pricingTypeEnum;
+        this.tuitionFee = tuitionFee;
+    }
+
+    public boolean canEnroll(int tuitionFee) {
+        if (pricingTypeEnum == PricingTypeEnum.FREE) {
```

Comment:
메시지를 보내 검증해도 좋을 것 같아요 
```suggestion
        if (pricingType.isFree()) {
```